src/system-health/health_checker/sysmonitor.py

```diff
@@ -0,0 +1,634 @@
+        syslog.syslog(syslog.LOG_CRIT, msg)
+
+        if also_print_to_console:
+            print(msg)
+
+    def log_alert(self, msg, also_print_to_console=False):
+        syslog.syslog(syslog.LOG_ALERT, msg)
+
+        if also_print_to_console:
+            print(msg)
+
+
+    def log_error(self, msg, also_print_to_console=False):
+        syslog.syslog(syslog.LOG_ERR, msg)
+
+        if also_print_to_console:
+            print(msg)
+
+    def log_warning(self, msg, also_print_to_console=False):
+        syslog.syslog(syslog.LOG_WARNING, msg)
+
+        if also_print_to_console:
+            print(msg)
+
+    def log_notice(self, msg, also_print_to_console=False):
+        syslog.syslog(syslog.LOG_NOTICE, msg)
+
+        if also_print_to_console:
+            print(msg)
+
+    def log_info(self, msg, also_print_to_console=False):
+        syslog.syslog(syslog.LOG_INFO, msg)
+
+        if also_print_to_console:
+            print(msg)
+
+    def log_debug(self, msg, also_print_to_console=False):
+        syslog.syslog(syslog.LOG_DEBUG, msg)
+
+        if also_print_to_console:
+            print(msg)
+
+
+
+#Initalise the syslog infrastructure
+logger = Logger(SYSLOG_IDENTIFIER)
+
+class Dict2Obj(object):
+    """dict to dict2obj
+    d: data"""
+
+    def __init__(self, d):
+        for a, b in list(d.items()):
+            if isinstance(b, (list, tuple)):
+                setattr(self, a, [Dict2Obj(x) if isinstance(
+                    x, dict) else x for x in b])
+            else:
+                setattr(self, a, Dict2Obj(b) if isinstance(b, dict) else b)
+
+
+class FileLock:
+   def __init__(self, lock_file):
+      self.f = open(lock_file, 'w')
+
+   def lock(self):
+      fcntl.flock(self.f, fcntl.LOCK_EX)
+
+   def unlock(self):
+      fcntl.flock(self.f, fcntl.LOCK_UN)
+      #self.f.close()
+
+sysready_lock = FileLock(SYSREADY_LOCKFILE)
+
+def update_proc_name(name):
+    try:
+        import setproctitle
+        setproctitle.setproctitle(name)
+        #pass
+    except Exception as err:
+        logger.log_error("Set proc name failed for {}: {}".format(name, str(err)))
+
+
+#Shows the system status message on the console and syslog 
+def show_system_status(state, msg):
+
+    logger.log_notice(msg)
+    with open('/dev/console', 'w') as console:
+        console.write("\n\n{} {} \n\n".format(datetime.datetime.now().strftime("%b %d %H:%M:%S.%f"), msg))
+
+
+def post_system_status(state, st_db):
+    try:
+        if st_db == None:
+            state_db = SonicV2Connector()
+            state_db.connect(state_db.STATE_DB, True)
+        else:
+            state_db = st_db
+        
+        state_db.set(state_db.STATE_DB, "SYSTEM_READY|SYSTEM_STATE", "Status", state)
+        logger.log_info("Posting system ready status {} to statedb".format(state))
+        
+    except Exception as e:
+        logger.log_error("Unable to post system ready status: {}".format(str(e)))
+
+
+def get_all_service_list(config_db):
```

Comment:
It is again a new process launch through run_command for systemctl command execution. Since it is easy to get the services from the specified directories, have used them.